changelog.txt

```diff
@@ -1,7 +1,17 @@
-4.3.1
+4.3.2
 =============
+New Features:
+- **MSAL.NET now sends telemetry data to the /token endpoint in regards to the error code of the previous request, if applicable**. This will enable MSAL.NET to determine reliablity across public client application calls.
+
 Bug Fixes:
+- **For iOS 13, iOS broker may or may not return the source application**. The source appplication is used by MSAL.NET to verify the response is coming from broker. MSAL.NET will now also create a nonce to send in the broker request and will verify the same nonce is returned in the broker response in the case of a missing source application. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1375)
+- **During a silent call, a token was issued for a given user for the initial tenant, and not the specified tenant**. MSAL.NET now uses the specified tenant. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1365)
+- **Claims are now sent to both the /authorize and /token endpoints**. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1328)
+
```

Comment:
***Experimental**
MSAL.NET now provides two extension methods enabling you to acquire an SSH certificate (to login to linux Azure VMs) interactively or silently.

Reply:
do you want this in a separate section? wasn't sure. 

---
In reply to: [325977767](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1392#discussion_r325977767) [](ancestors = 325977767)